src/emit.iced

```diff
@@ -113,18 +113,20 @@ exports.GoEmitter = class GoEmitter
     @emit_typedef_deep_copy { t }
     true
 
-  codec : ({name, optional, jsonkey}) ->
+  codec : ({name, optional, jsonkey, mpackkey}) ->
     omitempty = if optional then ",omitempty" else ""
     unless jsonkey?
       jsonkey = "#{name}"
-    "`codec:\"#{name}#{omitempty}\" json:\"#{jsonkey}#{omitempty}\"`"
+    unless mpackkey?
+      mpackkey = "#{name}"
```

Comment:
why not just `mpackkey = name`?

Reply:
because if it's set we want the value of mpackkey

Reply:
sorry, i meant why not:

```iced
unless mpackkey?
  mpackkey = name
```

Reply:
ohh idk I'll change them both